src/StackAdmin/AzureStack/AzureStack.psd1

```diff
@@ -51,7 +51,7 @@
     # ProcessorArchitecture = ''
 
     # Modules that must be imported into the global environment prior to importing this module
-    RequiredModules        = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.6.0'; },
+    RequiredModules        = @(@{ModuleName = 'AzureRM.Profile'; RequiredVersion = '5.5.2'; },
```

Comment:
@bganapa why is this change being made?

Reply:
I think that this might be because the 2.3.0 module has a RequiredVersion = 5.5.2.  We made a mistake there.

https://github.com/Azure/azure-powershell/blob/69981ef05c2e017e2833435755b8068bacc051cc/src/StackAdmin/AzureRM/AzureRM.psd1#L54

You can see it under the Package details as well.  I think Mark actually told us to do this, but that might have been another conversation.
https://www.powershellgallery.com/packages/AzureRM/2.3.0